src/sentry/api/endpoints/user_authenticator_details.py

```diff
@@ -16,7 +18,10 @@ class UserAuthenticatorDetailsEndpoint(UserEndpoint):
     def _get_device_for_rename(self, authenticator, interface_device_id):
         devices = authenticator.config
         for device in devices["devices"]:
-            if device["binding"]["keyHandle"] == interface_device_id:
+            if type(device["binding"]) == AuthenticatorData:
```

Comment:
i would add a comment here explaining that this is for webauthn and the other block is for old u2f devices that were registered.